DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -578,15 +578,18 @@ private void ConnectionManager_BannedFromChannel(object sender, ChannelEventArgs
 
             if (game == null)
             {
-                connectionManager.MainChannel.AddMessage(new ChatMessage(
-                    Color.White, "Cannot join channel " + e.ChannelName + ", you're banned!"));
-            }
-            else
-            {
-                connectionManager.MainChannel.AddMessage(new ChatMessage(
-                    Color.White, "Cannot join game " + game.RoomName + ", you've been banned by the game host!"));
+                var chatChannel = connectionManager.FindChannel(e.ChannelName);
+                if (chatChannel == null)
+                    return;
+
+                if(chatChannel.ChannelName != ProgramConstants.GeneralCncnetChatChannelName)
+                    connectionManager.MainChannel.AddMessage(new ChatMessage(Color.White, $"Cannot join chat channel {chatChannel.UIName}, you're banned!"));
```

Comment:
This line might be temporary, if the ban is removed from the IRC general chat.

Otherwise, this IF block contains "chat channel" specific language to use the chat channel's UI name, rather than the #cncnet channel name, which is I believe what's confusing people. They think they're banned from cncnet as a whole, not just a chat channel.